client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/email-provider-stacked-features.tsx

```diff
@@ -50,4 +50,28 @@ const EmailProviderStackedFeatures: FunctionComponent< EmailProviderStackedFeatu
 	);
 };
 
-export default EmailProviderStackedFeatures;
+export interface EmailProviderStackedFeaturesProps {
+	features: TranslateResult[];
+}
```

Comment:
This interface is already declared and exported on L27-29, so these lines should be removed.
```suggestion
```